package.json

```diff
@@ -35,6 +35,7 @@
     "@ngrx/effects": "^2.0.3",
     "@ngrx/store": "^2.2.2",
     "angular2-jwt": "0.1.25",
+    "cordova-custom-config": "4.0.2",
```

Comment:
From npm: [why should I use cordova-custom-config](https://www.npmjs.com/package/cordova-custom-config#why-should-i-use-it)

I don't know exactly what installed it, but I know that npm installs it every time I run `npm install`.